From acae6bf4594f80da57855343ab325f87386178c4 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Fri, 3 Nov 2023 16:40:54 +0100
Subject: [PATCH] tests: Fix targetcli_config.json

Not all attributes are available anymore in newer kernel versions.
---
 src/tests/dbus-tests/targetcli_config.json | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/tests/dbus-tests/targetcli_config.json b/src/tests/dbus-tests/targetcli_config.json
index 3be9eac2be..f50bf7d4d2 100644
--- a/src/tests/dbus-tests/targetcli_config.json
+++ b/src/tests/dbus-tests/targetcli_config.json
@@ -331,7 +331,6 @@
             "demo_mode_write_protect": 0,
             "generate_node_acls": 1,
             "login_timeout": 15,
-            "netif_timeout": 2,
             "prod_mode_write_protect": 0,
             "t10_pi": 0,
             "tpg_enabled_sendtargets": 1
@@ -393,7 +392,6 @@
             "demo_mode_write_protect": 1,
             "generate_node_acls": 0,
             "login_timeout": 15,
-            "netif_timeout": 2,
             "prod_mode_write_protect": 0,
             "t10_pi": 0,
             "tpg_enabled_sendtargets": 1
@@ -479,7 +477,6 @@
             "demo_mode_write_protect": 1,
             "generate_node_acls": 0,
             "login_timeout": 15,
-            "netif_timeout": 2,
             "prod_mode_write_protect": 0,
             "t10_pi": 0,
             "tpg_enabled_sendtargets": 1
